gobblin-data-management/src/main/java/gobblin/data/management/copy/extractor/CloseableFsFileAwareInputStreamExtractor.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.data.management.copy.extractor;
+
+import gobblin.data.management.copy.CopyableFile;
```

Comment:
I am using Linkedin's eclipse formatter that has gobblin, java, org, com. Does gobblin have a different formatter xml i need to use?